CRM/Contribute/BAO/Contribution.php

```diff
@@ -5457,4 +5457,25 @@ public static function createProportionalFinancialEntries($entityParams, $lineIt
     }
   }
 
+  /**
+   * Calculate Tax for each item when Financial Type is changed.
+   *
```

Comment:
This function is just repeatedly calling a different function that does the calculation. I don't think it should be documenting details that is not responsible for.